app/views/service_requests/service_request_assignments/_add_card.html.slim

```diff
@@ -0,0 +1,49 @@
+- if !@account.have_payment_method
+  .payment_methods
+    div
+      = link_to 'Default Payment Method', '#', class: 'btn btn-default w100-pct text-center mt10'
+    div
+      = link_to 'Add New Card', service_requests_service_request_assignment_start_accepting_path(assignment.id, service_request_id: service_request.id), remote: true, class: 'btn btn-default w100-pct text-center mt10'
+    div
+      = link_to 'Done', '#', remote: true, class: 'btn btn-danger w100-pct text-center mt20'
+- else
+  .panel-body
+    .text-center
+      span.visa
+        = image_tag('visa.png', size: '58X94')
+      span.master_card
+        = image_tag('masterCard.png', size: '58X94')
+      span.american_express
+        = image_tag('ae.png', size: '58X94')
+    .card-bg.mt15
+      .card-front
+        .text-right
+          .card-image.visa-img
+        .row.card-details
+          .col-md-9
+            label Card number
```

Comment:
Please add attribute `for` with id of related field, that's a native HTML feature.